qiita_db/util.py

```diff
@@ -506,13 +506,15 @@ def compute_checksum(path):
     return crc & 0xffffffff
 
 
-def get_files_from_uploads_folders(study_id):
+def get_files_from_uploads_folders(study_id, ignore_hidden_and_folders=True):
```

Comment:
Would it be better to have this as two bools, one for hidden and one for folders? Would be more useful in the long term, especially if we start allowing folder uploads.